src/cpp/server/load_reporter/load_reporter_async_service_impl.cc

```diff
@@ -358,7 +358,7 @@ void LoadReporterAsyncServiceImpl::ReportLoadHandler::Shutdown(
 }
 
 void LoadReporterAsyncServiceImpl::ReportLoadHandler::OnFinishDone(
-    std::shared_ptr<ReportLoadHandler> self, bool ok) {
+    const std::shared_ptr<ReportLoadHandler>& self, bool ok) {
```

Comment:
This change doesn't look right to me.  I think this function is supposed to release the last ref when it finishes.

Reply:
The one holding the last ref is `CallableTag` so technically this would work but I like to make this function consistent with other handlers in the class. So I'm rather reverting this and adding NOLINT comment to the function.